authentication/account/service/user_blackbox_test.go

```diff
@@ -140,6 +143,11 @@ func (s *userServiceBlackboxTestSuite) TestListUsersToNotifyBeforeDeactivation()
 		require.NoError(s.T(), err)
 		require.NotNil(s.T(), identity.DeactivationNotification)
 		assert.True(s.T(), time.Now().Sub(*identity.DeactivationNotification) < time.Second*2)
+		// also verify that the message to send to the user has the correct data
+		assert.Equal(s.T(), identity2.ID.String(), msgToSend.TargetID)
+		expiryDate := time.Now().Add(config.GetUserDeactivationInactivityPeriodDays() - config.GetUserDeactivationInactivityNotificationPeriodDays()).Format("Mon Jan 2")
```

Comment:
finding expected value using time.Now() and actual value also use time.Now() will fail if test has started about `11:59:59 PM` and end about `12:00:00 AM`.  In this case there is chances that the test will fail because two different days will be compared.  I am talking about very rare possibility.  The point is, calling time.Now() two times is not right way to compare actual and expected values.

Reply:
ok, let me address that

Reply:
done in 4893646